VimeoUpload/UIKit/Extensions/NSString+Conversions.swift

```diff
@@ -61,21 +61,21 @@ public extension NSString
         }
 
         var string = String(format: "%.1f %@", value, suffix)
-        if let range = string.rangeOfString(".0")
+        if let range = string.range(of: ".0")
         {
-            string.replaceRange(range, with: "")
+            string.replaceSubrange(range, with: "")
         }
         
-        return string
+        return string as NSString
     }
     
-    static func stringFromDurationInSeconds(duration: Float64) -> NSString
+    static func stringFromDuration(inSeconds duration: Float64) -> NSString
     {
         let duration = max(duration, 0)
         
         let hours = Int(floor(duration / (60 * 60)))
-        let minutes = hours > 0 ? Int(floor((duration % (60 * 60)) / 60)) : Int(floor(duration / 60))
-        let seconds = Int(floor(duration % 60))
+        let minutes = hours > 0 ? Int(floor((duration.truncatingRemainder(dividingBy: (60 * 60))) / 60)) : Int(floor(duration / 60))
```

Comment:
are we positive that truncatingRemainder is functionally equivalent to using the modulus operator?